packages/perseus/src/widgets/number-line/number-line.tsx

```diff
@@ -191,8 +199,8 @@ const TickMarks: any = Graphie.createSimpleClass((graphie, props) => {
 
 // TODO: most widgets use some like Widget<Something, PerseusNumberLineWidgetOptions>
 // should this one?
-type Props = ChangeableProps & {
-    range: [number, number];
+type Props = {
+    range: number[];
```

Comment:
I thought `[number, number]` was preferred when there was a fixed number of elements?

Reply:
The `data-schema` types have `range: number[]`, and the parser (which is derived from data-schema) does too. Since we apparently don't guarantee that there will always be two elements, I opted to change this type to be compatible with data-schema rather than vice-versa.